VTEX - Checkout API.json

```diff
@@ -644,15 +717,50 @@
+                                                                        ],
+                                                                        "properties": {
+                                                                            "name": {
+                                                                                "type": "string",
+                                                                                "description": "Name of the attachment. Ex. `vtex.subscription.weekly`.",
+                                                                                "nullable": true
+                                                                            },
+                                                                            "required": {
+                                                                                "type": "boolean",
+                                                                                "description": "If the attachment is required (`true`) or not (`false`).",
+                                                                                "nullable": true
+                                                                            },
+                                                                            "schema": {
+                                                                                "type": "object",
+                                                                                "description": "Custom values [created into attachment](https://help.vtex.com/en/tutorial/adding-an-attachment).",
+                                                                                "nullable": true
+                                                                            }
+                                                                        }
+                                                                    }
+                                                                },
                                                                 "shippingEstimate": {
                                                                     "type": "string",
                                                                     "description": "Shipping estimate. For instance, \"three business days\" will be represented as `3bd`."
                                                                 },
                                                                 "shippingEstimateDate": {
                                                                     "type": "string",
-                                                                    "description": "Shipping estimate date.",
+                                                                    "description": "When using the query parameter `individualShippingEstimates=true`, it will contain the estimated shipping date (ex. `2023-09-09T11:29:00+00:00`), otherwise it will contain `null`.",

```

Comment:
Theoretically there's no need to add the example here, since it will already be in the response body example. If you do choose to keep it, please replace "ex" with "e,g," everywhere where this appears